src/discussions/post-comments/PostCommentsView.jsx

```diff
@@ -29,7 +29,7 @@ const PostCommentsView = () => {
   const intl = useIntl();
   const navigate = useNavigate();
   const location = useLocation();
-  const isOnDesktop = useIsOnDesktop();
+  const isOnTablet = useIsOnTablet();
```

Comment:
use same name for consistency 
instead of "isOnTablet" use "isOnTabletorDesktop"